### PR TITLE
fix errors in vue template when run yarn serve

### PR DESCRIPTION
**Pull request by personball**
Downgrade iview version to 2.13.1, and vue-property-decorator to 6.0.0, can fix errors when run yarn serve in vue template.

### CODE REVIEW

File: vue/package.json [hunk 1]
```diff
@@ -11,12 +11,12 @@
     "abp-web-resources": "^3.8.2",
     "axios": "^0.18.0",
     "famfamfam-flags": "^1.0.0",
-    "iview": "^3.0.0",
+    "iview": "^2.13.1",
```
Comment (acjh) [inline vue/package.json]: `~3.0.0` is fine. See https://github.com/aspnetboilerplate/module-zero-core-template/issues/343#issuecomment-437894647.

Comment (personball) [reply vue/package.json]: Not release yet?

Comment (acjh) [reply vue/package.json]: Not changed yet, as I was hoping for a fix in iView.
You can modify your PR to make that change instead.
It's preferable as compared to downgrading a major version.

Remember to update that line in **yarn.lock**. Thanks!

**Comment by personball** [comment]
@acjh  Wait, you mean just modify one line of iview version ^3.0.0 to ~3.0.0?

**Comment by personball** [comment]
@acjh  It works, Thanks.

**Comment by ismcagdas** [comment]
@personball is your PR completed ? If so, we will review and merge it.

**Comment by personball** [comment]
@ismcagdas Yes.

**Pull request merged**